content/articles/2018-10-devtools-2.0.0.Rmarkdown

```diff
@@ -0,0 +1,228 @@
+---
+title: devtools 2.0.0
+date: 2018-10-26
+slug: devtools-2-0-0
+author: Jim Hester
+categories: [package]
+description: >
+    Make package development easier by providing R functions that simplify and expedite common tasks.
+photo:
+  url: https://unsplash.com/photos/wEJK4q_YlNQ
+  author: Hunter Haley
+---
+
+## Introduction
+
+[devtools](https://cran.r-project.org/package=devtools) 2.0.0 is now on CRAN.
```

Comment:
We usually link to the package website from the first mention of its name

Reply:
Should this link be to the CRAN page or to <http://devtools.r-lib.org>? I favour the latter.

Reply:
Sorry, please ignore me. Looking at outdated version 🤪